src/js/flows/initSpace.js

```diff
@@ -56,6 +56,7 @@ function getCurrentSpaceName(spaces, desired) {
 function setSpace(dispatch, data, clusterId) {
   globalDispatch(Spaces.setDetail(clusterId, data))
   dispatch(Search.setSpace(data.name))
+  data = brim.interop.spacePayloadToSpace(data)
```

Comment:
@jameskerr  The problem is that the span parsing we are doing rn is late in the stack, in the reducer (https://github.com/brimsec/brim/pull/715/files#diff-31bf947bfb654ba8ccfd57214bfb417dR21), and yet the original span which has not been put through that adapter/parser continues to get passed along by this function to the rest of the initSpace chain. So I've adjusted this to do the same operation on a different copy and return that. Not a great pattern, would be better if we can handle this mutation inside the client, or at least much earlier before it gets passed around in different formats.